observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+TLDR - Dashboard to show patterns. What you want to achieve here is to derive a pattern - user, usage, search or anything that you have defined as your KPI.
+
+## Best Practices
+
+1. Identify your goal or KPI.
+2. Ask yourself questions that can help you reach your goal or KPI. This may sound counter intuitive, the more questions you ask the better the outcome will be.
+3. Validate your questions. Are these necessary in finding a common pattern?
+4. Build simple chart, have more of them. Rather than having a complicated all in one dashboard, seperate your dashboard into sections. Azure Workbooks is highly recommended as it supports markdown.
+5. Observe the dashboard that you built. Is the data reflecting what you are set out to answer? If not make small adjustment.
+
+Always remember this process takes time. Building dashboard is easy, building an obserable dashboard to show pattern is hard.
+
+## Recommended Tools
+
+- [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview) - Umbrella of services including system metrics, log analytics and more.
+- [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) - An extensible Application Performance Management (APM) service used to monitor live applications, and a feature of inside Azure Monitor.
+- [Azure Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview) - Run advanced queries against the data collected via Azure Monitor Logs.
+- [Azure Monitor Workbooks](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/workbooks-overview) - Visualizations for your logged data, tightly interated with Azure services and higly customizable.
+
```

Comment:
Perhaps add a non-Microsoft monitoring tool here?

Reply:
+1, as these 4 items are really the same product.

Reply:
Grafana is a good candidate to have on the list (it's also listed in the other pillars)

Reply:
added quick starts and not merely tools